ipaserver/plugins/baseldap.py

```diff
@@ -715,7 +715,9 @@ def get_memberofindirect(self, entry):
             result = self.backend.get_entries(
                 self.api.env.basedn,
                 filter=filter,
-                attrs_list=[''])
+                attrs_list=[''],
+                size_limit=-1, # paged search will get everything anyway
```

Comment:
`./ipaserver/plugins/baseldap.py:719:31: E261 at least two spaces before inline comment`
For some reason Travis did not report this, I may need to check why.